benchto-driver/src/main/java/com/teradata/benchto/driver/BenchmarkProperties.java

```diff
@@ -31,16 +31,18 @@
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static com.google.common.base.Preconditions.checkState;
 import static com.teradata.benchto.driver.utils.PropertiesUtils.splitProperty;
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toMap;
 
 @Component
 public class BenchmarkProperties
 {
     @Value("${sql}")
-    private String sqlDir;
+    private String sqlDirs;
 
     @Value("${benchmarks}")
-    private String benchmarksDir;
+    private String benchmarksDirs;
```

Comment:
Let it be next step if required. This way we don't break any scripts that rely on those two arguments to be separate.